src/SFA.DAS.Roatp.ProviderModeration.Web/Views/ProviderSearch/ProviderDescriptionReview.cshtml

```diff
@@ -18,9 +18,8 @@
             <input type="hidden" asp-for="@Model.LegalName" />
             <input type="hidden" asp-for="@Model.ProviderDescription" />
             <div esfa-validation-marker-for="ProviderDescription" class="govuk-form-group">
-            <label class="govuk-label" asp-for="ProviderDescription">
-                @Model.ProviderDescription
-            </label>
+            <p style="white-space: pre-line">@Model.ProviderDescription</p>
```

Comment:
Add the style to `wwwroot/css/app.css` as a class to `P` tag and apply the class in the views.